tcp-native/src/main/java/org/ballerinalang/stdlib/tcp/SslHandshakeClientEventHandler.java

```diff
@@ -0,0 +1,49 @@
+package org.ballerinalang.stdlib.tcp;
+
+import io.ballerina.runtime.api.Future;
+import io.netty.channel.ChannelHandlerContext;
+import io.netty.channel.ChannelInboundHandlerAdapter;
+import io.netty.handler.flow.FlowControlHandler;
+import io.netty.handler.ssl.SslHandshakeCompletionEvent;
+import org.ballerinalang.stdlib.tcp.nativeclient.Client;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Class to handle SSL handshake event of TCP Client.
+ */
+public class SslHandshakeClientEventHandler extends ChannelInboundHandlerAdapter {
+    private TcpClientHandler tcpClientHandler;
+    private Future callback;
```

Comment:
I think the better name would be `balInitCallback`

Reply:
updated